### ISSUE TITLE
unable to build on WSL

### ISSUE THREAD

**Issue by kibabyte** [sandydoo/flux-screensavers#18]
![grafik](https://user-images.githubusercontent.com/97415279/229655134-54a4b0f2-5021-4cc0-9a70-402ff56f043e.png)
This is as far is I can get with the information given here. I am totally new to nix and confused on what exactly I am supposed to do here. Any hint in the right direction would be greatly appreciated!

**Comment by HadiJadallah** [comment]
Hello,

This is how I got it to compile on my WSL 2 Windows 11 Ubuntu 20.04
Prepare Environment:
1- git clone the main repo
2- Install rust using: curl --proto '=https' --tlsv1.3 https://sh.rustup.rs -sSF | sh
2.1 First install as default
2.2 run again and use custom to insall x86_64-pc-windows-gnu target
2.3 follow on screen instruction
3. apt install build-essential package
4. follow instructions on https://nixos.org/download.html#nix-install-linux  to install nix per your environment
5. close and open a new shell to release all the variables
6. rustup target add x86_64-pc-windows-gnu
7. I had to install libsdl2-dev but when the next nix build command it downloaded again
8. nix build --extra-experimental-features flakes _--extra-experimental-features nix-command

and it compiled for me
hope it helps
